extensions/mssql/src/features.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import { SqlOpsFeature, SqlOpsDataClient } from 'dataprotocol-client';
+import { ClientCapabilities, ServerCapabilities, RPCMessageType, StaticFeature } from 'vscode-languageclient';
+
+import { Disposable } from 'vscode';
+
+import { Telemetry } from './telemetry';
+import * as Utils from './utils';
+import { TelemetryNotification } from './contracts';
+
+class TelemetryFeature implements StaticFeature {
```

Comment:
This class isn't being used. Can we remove it?

Reply:
It should be being used..hmm